Dockerfile

```diff
@@ -38,6 +38,7 @@ ENV UVICORN_PORT=8000
 
 RUN apt update && \
   apt-get install --yes --no-install-recommends \
+    mariadb-client-core \
```

Comment:
nit(non-blocking): is this a requirement? can't see any other code related to mariadb